content/post/announcing-pga.md

```diff
@@ -0,0 +1,129 @@
+
+## Public Git Archive Schema
+
+Public Git Archive provides an index file with information that can be used to select which parts
+of the dataset one might want to download. This includes:
+
+- repository URL,
+- name of the [siva](https://github.com/src-d/siva) with the repository contents
+- number of total files, lines of code, and bytes,
+- number of total commits and commits in the HEAD path to the root,
+- list of languages detect in the repository with [enry](https://github.com/src-d/enry),
+- license detected in the repository with [go-license-detector](https://github.com/src-d/go-license-detector).
+
+You can find the [documented schema](https://pga.sourced.tech/) of the dataset on GitHub.
+
+## Download it now!
+
+You can download [the index file](http://pga.sourced.tech/csv/latest.csv.gz), find the `siva`
+files corresponding to the repositories of your interested, and download one by one.
+
+To make this task easier we've also released a tool that we call `pga`. `pga` is able to filter
+and download sections of the datasets with an easy command line interface.
+You can download it from [the releases page at github.com/src-d/datasets](https://github.com/src-d/datasets/releases).
+Alternatively, if you already have Go installed on your machine, you can simply run
+`go get github.com/src-d/datasets/PublicGitArchive/pga`.
+
+Once installed listing all of the repositories in the dataset is as simple as running `pga list`.
+To list only repositories that contained some Java:
+
+```bash
+▶️ pga list -l java
+https://github.com/karlisson/1001
+https://github.com/thiagolocatelli/android-uitableview
+https://github.com/JakeWharton/Android-ViewPagerIndicator
+...
+```
+
+, or to show only the repositories under the `src-d` organization:
+
+```bash
+▶️ pga list -u /src-d/
+https://github.com/src-d/beanstool
+https://github.com/src-d/kmcuda
+https://github.com/src-d/hercules
+...
+```
+
+To download the data, rather than just listing the repositories, replace `list` with `get`.
+
+Finally, you can also use `pga` to present the index in different formats, such as `CSV` or `JSON`:
+
+This can be useful when a more complex filter of the index is necessary.
+For instance, if you wanted to download all of the source{d} repositories with Apache v2 licenses,
+you can run the following command:
+
+```bash
+▶️ pga list -u /src-d/ -f json | grep "Apache-2.0" | jq -r ".sivaFilenames[]" | pga get -i
+downloading siva files by name from stdin
+filter flags will be ignored
+ 0 / 5 [------------------------------------------------------]   0.00%
+```
+
+We first list all of the source{d} repositories as JSON, grep those using Apache v2 licenses,
+then use [jq](https://stedolan.github.io/jq/) to extract the corresponding filenames, and download them with `pga get -i`.
+
+## License
+
+`Public Git Dataset` is available under the
+[Open Database License (ODBL)](https://opendatacommons.org/licenses/odbl/),
```

Comment:
This is the license of the index file, we cannot license the code itself because we do not own it. There is a grey area for those 30% which do not contain any license information. We must mention that the dataset purpose is **doing research only** - otherwise, we violate the GitHub ToS.

Besides, it is dual licensed with CC-BY-SA-NC 4.0 - the paper points to that license.

Reply:
Fixed